pkg/scheduler/framework/plugins/nodeports/node_ports.go

```diff
@@ -97,6 +98,16 @@ func getPreFilterState(cycleState *framework.CycleState) (preFilterState, error)
 	return s, nil
 }
 
+// EventsToRegister returns the possible events that may make a Pod
+// failed by this plugin schedulable.
+func (pl *NodePorts) EventsToRegister() []framework.ClusterEvent {
+	return []framework.ClusterEvent{
+		{Resource: framework.Pod, ActionType: framework.Delete},
+		{Resource: framework.Node, ActionType: framework.Add},
+		{Resource: framework.Service, ActionType: framework.Delete},
```

Comment:
why Service?

Reply:
@alculquicondor Thank you for your correction.
I learned the relevant knowledge, and finally figured out the difference between "port" for pods and services. I have deleted this wrong event. Currently I am testing the addition and deletion of nodes and pods.
Thanks again for your help.